mlir/lib/Quantum/Transforms/emit_catalyst_pyface.cpp

```diff
@@ -142,7 +142,8 @@ void wrapResultsAndArgsInTwoStructs(LLVM::LLVMFuncOp op, PatternRewriter &rewrit
         /*cconv*/ LLVM::CConv::C);
 
     OpBuilder::InsertionGuard guard(rewriter);
-    rewriter.setInsertionPointToStart(wrapperFuncOp.addEntryBlock());
+    auto entryBlock = wrapperFuncOp.addEntryBlock(rewriter);
+    rewriter.setInsertionPointToStart(entryBlock);
```

Comment:
Was this causing memory issues to not keep the variable around? 👀

Reply:
They changed the interface of `wrapperFuncOp.addEntryBlock`. In the new version it must takes `rewriter` as an argument.